frontend/src/app/modules/matches/matches-saga.test.js

```diff
@@ -15,13 +15,28 @@ const ERROR = {
 describe('matchesSaga', () => {
   describe('addMatchSaga', () => {
     let gen
-    const expectNextToBe = (value, done = false) => {
-      const result = gen.next()
-      expect(result.value).toStrictEqual(value)
-      expect(result.done).toBe(done)
+    const expectNext = (suppliedValue, generated) => {
+      expect(gen.next(suppliedValue)).toStrictEqual(generated)
```

Comment:
i'd call this "expected" not generated

Reply:
agree, thanks, fixed in https://github.com/salsita/foosball-rating/pull/125/commits/4a803ce91ad0348134a945f4c581076d76e339aa